js/background.js

```diff
@@ -145,7 +145,13 @@ chrome.runtime.onMessage.addListener((message, sender, sendResponse) =>
 			chrome.tabs.create({ url: message.url });
 			break;
 		case "loadData":
-			sendResponse(collections);
+			chrome.storage.sync.get("sets", values =>
+			{
+				if (values?.sets != null)
+					collections = JSON.parse(values?.sets)
+				sendResponse(collections);
+				UpdateTheme();
+			});
 			break;
```

Comment:
The sendResponse is returned asynchronously in a callback, so we would have to return true according to [these docs](https://developer.chrome.com/extensions/messaging)
```suggestion
					collections = JSON.parse(values?.sets)
				sendResponse(collections);
				UpdateTheme();
			});
			return true; //Required to indicate the answer will be sent asynchronously https://developer.chrome.com/extensions/messaging
			break;
```